### PR TITLE
Omkar first issue

### PR DESCRIPTION
**Pull request by omkarbasnet1**

### CODE REVIEW

File: evennia/utils/tests/test_utils.py [hunk 1]
```diff
@@ -54,6 +82,95 @@ def test_dedent(self):
         self.assertEqual(expected_string, utils.dedent(input_string))
 
 
+from django.test import TestCase
+from evennia.utils.utils import class_from_module
```
Comment (Griatch) [inline evennia/utils/tests/test_utils.py:86]: This should not be re-imported again in the middle of the module.

**Review by Griatch — CHANGES_REQUESTED** [review]
Thanks for contributing! Based on emails, was this some sort of class contribution? :) You must write a better title and description when making a PR, or it easily looks like a mistake posting.

Alas, you are testing functionality that already have extensive tests, and there are  code quality issues here too, like re-importing in the middle of tests. This can't merge.

The `TestClassFromModule` test does add new testing, so that could be useful, but the rest appear to be duplicates, alas. So please update the PR to only include the code adding to the code coverage, fixing the python code.

**Review by Griatch — CHANGES_REQUESTED** [review]
Sorry, you cannot keep adding new fixes to the same branch. Make separate branches/PRs for your features. Since there is no sign of responding to feedback, I'm closing this PR - please open new, separate PRs for these fixes.
